query/autocomplete_defaults.js

```diff
@@ -20,6 +20,7 @@ module.exports = _.merge({}, peliasQuery.defaults, {
   'ngram:field': 'name.default',
   'ngram:boost': 100,
   'ngram:cutoff_frequency': 0.01,
+  'ngram:minimum_should_match': '1<-1 3<-25%',
```

Comment:
This setting was used just because it's what we've used before. The explanation of what it does and why is contained in https://github.com/pelias/api/pull/1333